airflow/settings.py

```diff
@@ -273,7 +273,7 @@ def configure_orm(disable_connection_pool=False, pool_class=None):
 
 DEFAULT_ENGINE_ARGS = {
     "postgresql": {
-        "executemany_mode": "values",
+        "executemany_mode": "values_only",
```

Comment:
Does 1.4 support `values_only`?

Reply:
Yep, `values` even not listed in allowed values in 1.4.
But as I could understand the `values` -> `values_plus_batch `

https://github.com/sqlalchemy/sqlalchemy/issues/5401

Reply:
In that case maybe we don't need to raise the lower version bound, and can make the version range `>1.4.36,<2.1.0` instead?

Reply:
Ahhh... I miss this change. Yeah we definitely should keep support of SA 1.4 as much as possible, not all providers could work with SA 2.0

But also I'm not sure that we need to add support SA 2.0, because I'm not totally sure that Airflow could work with it properly.

Reply:
> Ahhh... I miss this change. Yeah we definitely should keep support of SA 1.4 as much as possible, not all providers could work with SA 2.0
> 
> But also I'm not sure that we need to add support SA 2.0, because I'm not totally sure that Airflow could work with it properly.

Agreed. The FAB library you mentioned earlier is required to update their SA. I'll have a quick look on what is needed to update that dependency.

Reply:
FAB is being migrated in https://github.com/apache/airflow/pull/38319 -but still sqlalchemy is < 1.5 there.